xml/src/main/java/org/ehcache/xml/ConfigurationParser.java

```diff
@@ -598,6 +587,15 @@ private static MemoryUnit parseMemory(MemoryType memoryType) {
     return xmlConfigurationParser.parseServiceConfiguration(element);
   }
 
+  ResourcePool parseResourceExtension(final Element element) {
+    URI namespace = URI.create(element.getNamespaceURI());
+    final CacheResourceConfigurationParser xmlConfigurationParser = resourceXmlParsers.get(namespace);
+    if (xmlConfigurationParser == null) {
+      throw new XmlConfigurationException("Can't find parser for namespace: " + namespace);
+    }
```

Comment:
Again, I don't see the point ... the `if` in this case is checking for an exceptional condition.  You'd apply this _everywhere_ an if is used?